contracts/src/v0.8/automation/dev/chains/OptimismModule.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.19;
+
+import {OVM_GasPriceOracle} from "./../../../vendor/@eth-optimism/contracts/v0.8.9/contracts/L2/predeploys/OVM_GasPriceOracle.sol";
+import {IChainModule} from "../interfaces/v2_2/IChainModule.sol";
+
+contract OptimismModule is IChainModule {
```

Comment:
I think we can make the modules inherit `ChainModuleBase`, and then we only have to override the functions that are _different_ ex in the OP module we can remove the blockNumber() and blockHash() functions

Reply:
done